### ISSUE TITLE
Requirements for Software Security Engineering Assignment

### ISSUE THREAD

**Issue by Lord-Tiger** [Lord-Tiger/CYBR8420_Fall24#7]

**Comment by Lord-Tiger** [comment]
![Admin_Use_ _Misuse_Cases](https://github.com/user-attachments/assets/60ade503-587f-4bb2-a28d-0713d8d737ab)
My input for us to discuss.

**Comment by asongne** [comment]
It looks good to me

**Comment by Lord-Tiger** [comment]
![Admin_Use_ _Misuse_Cases](https://github.com/user-attachments/assets/68609653-af1c-4fad-bfa5-c2f70af4535f)
![Patient_Use_ _Misuse_Cases](https://github.com/user-attachments/assets/759dcccf-7160-412d-84aa-f512816b55c3)

**Comment by Lord-Tiger** [comment]
![Admin_Use_ _Misuse_Cases2](https://github.com/user-attachments/assets/dc5081c6-b9a7-426a-b443-67c06fdc9b31)

**Comment by Lord-Tiger** [comment]
![Admin_Use_ _Misuse_Cases2](https://github.com/user-attachments/assets/fd1494ba-3160-458e-8f67-ce2a06f9449e)

**Comment by gilbeb** [comment]
![Image](https://github.com/user-attachments/assets/19eb69b8-ce43-4ffd-85a5-c29b2142a6ad)

**Comment by asongne** [comment]
This project was the most challenging we have undertaken. Our initial difficulty was grasping the core problem we needed to address. In our first attempt at building a misuse case, we mistakenly used three different actors for a single case, when a simpler approach involving just one actor—such as a basic login scenario—would have been more appropriate. After consulting with our professor, we gained some direction; however, there was still some ambiguity for me personally. Despite this, we managed to develop three misuse cases and requested each team member to create a diagram and upload it to our GitHub repository for review and feedback from the rest of the group.
